src/main/java/com/lewisd/maven/lint/rules/AbstractReduntantVersionRule.java

```diff
@@ -47,20 +47,20 @@ protected void checkForRedundantVersions(final MavenProject mavenProject,
         }
 
         String version = resolveVersion(modelObject, resolvedModelObject);
-        String inheritedVersion = modelUtil.getVersion(tryResolveObject(inheritedObject));
+        String inheritedVersion = getModelUtil().getVersion(tryResolveObject(inheritedObject));

```

Comment:
sure Liskov means extends classes behave like theirs parents and java's object orientation is used with accessors. So avoiding the usage of `accessors` bring encapsulation in danger. And broken encapsulation brings refactoring in danger.
If everything is opened you have a tight coupled system and nothing can be changed without changing all.

So ... yes you can do ... but in the long run, your code becomes a big mess. It is on you!